modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/validations.form.js

```diff
@@ -40,6 +41,24 @@ const isLowercaseAndNumbers = (value) => {
 	}
 };
 
+const isStartingWithSlash = (value) => {
```

Comment:
```suggestion
const startsWithSlash = (value) => {
```

Reply:
I already made Jair suggestion's here and create just one validation for all the URL validations. Thanks, Iago!